guides/v2.3/graphql/mutations.md

```diff
@@ -0,0 +1,136 @@
+---
+group: graphql
+title: Mutations
+---
+
+While GraphQL queries perform read operations, mutations change the data. A mutation can create, update, or delete objects and fields. In REST terminology, queries operate like `GET` requests, while mutations are similar to `POST`, `PUT`, and `DELETE`.
```

Comment:
This is a great description of a mutation. I'll link to this topic from my customer topic where I briefly describe mutations.